extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantFeatureFinder.java

```diff
@@ -35,32 +36,51 @@ public static TokenCustomizer find(OidcTenantConfig oidcConfig) {
                     throw new OIDCException("Unable to find TokenCustomizer " + customizerName);
                 }
             } else if (oidcConfig.tenantId.isPresent()) {
-                return container
-                        .instance(TokenCustomizer.class, TenantFeature.TenantFeatureLiteral.of(oidcConfig.tenantId.get()))
-                        .get();
+                String tenantId = oidcConfig.tenantId.get();
+                List<TokenCustomizer> list = findTenantFeaturesByTenantId(TokenCustomizer.class, tenantId, container);
+                if (!list.isEmpty()) {
```

Comment:
@Eng-Fouad I've only noticed it now, I think it is fine to have multiple features of the same type applied to the same tenant, so propose to drop this check

Reply:
@Eng-Fouad Oh sorry, does it not return a List... I'd fix it to return a List. Let me look a bit later into it